### DISCUSSION TITLE
Still rejected

### DISCUSSION THREAD

**Discussion by Bhavigit** [community/community#37076] [GitHub Education]
I have submitted the 5 requests but all are rejected and again i will create a new GitHub account and also create new mail account  and again submit the New request on yesterday 10 oo clock still complete 24 hours but the response is not come it's on pending process which time it will be taken how  I can solve this problem... pls suggest anyone it's a urgent

**Comment by rskbansal** [comment]
Are you uploading the right documents? Try uploading a proof of you being a student, maybe a ID provided by you University or any other document which proves that you are a student at that place, maybe like an admission slip
